crates/test-programs/wasi-http-tests/src/bin/outbound_request.rs

```diff
@@ -111,7 +115,7 @@ fn main() -> Result<()> {
 
     println!("localhost:3000 /get: {r1:?}");
     assert_eq!(r1.status, 200);
-    let method = findHeader(&r1, "x-wasmtime-test-method".to_string()).unwrap_or("MISSING".to_string());
+    let method = r1.header("x-wasmtime-test-method").unwrap_or(&missing);
```

Comment:
Tests are the one place in Rust I prefer to write code that panics on failure - after all, that is what assert_eq! is doing. So in this case I would write this as `let method = r1.header("x-wasmtime-test-method").unwrap();` rather than mess around with having the alternative value around with the correct type, which is clunky.

Reply:
Thanks, I'm still learning idiomatic rust. I will correct this in a follow-on PR.